drivers/scheduler/sidecars/es-stress-test/elasticsearch-stress-test.py

```diff
@@ -0,0 +1,462 @@
+            print(e)
+            sys.exit(1)
+
+        # Generate docs
+        documents_templates = generate_documents()
+        fill_documents(documents_templates)
+
+        print("Done!")
+        print("Creating indices.. ")
+
+        indices = generate_indices(es)
+        all_indices.extend(indices)
+
+        try:
+            #wait for cluster to be green if nothing else is set
+            if WAIT_FOR_GREEN:
+                es.cluster.health(wait_for_status='green', master_timeout='600s', timeout='600s')
+        except Exception as e:
+            print("Cluster timeout....")
+            print("Cleaning up created indices.. "),
+
+            cleanup_indices(es, indices)
+            continue
+
+        print("Generating documents and workers.. ")  # Generate the clients
+        clients.extend(generate_clients(es, indices, STARTED_TIMESTAMP))
+
+        print("Done!")
+
+
+    print("Starting the test. Will print stats every {0} seconds.".format(STATS_FREQUENCY))
+    print("The test would run for {0} seconds, but it might take a bit more "
+          "because we are waiting for current bulk operation to complete. \n".format(NUMBER_OF_SECONDS))
+
+    # Run the clients!
+    for d in clients:
+        d.start()
+
+    # Create and start the print stats thread
+    stats_thread = Thread(target=print_stats_worker, args=[STARTED_TIMESTAMP])
+    stats_thread.daemon = True
+    stats_thread.start()
+
+    for c in clients:
+       while c.is_alive():
+            try:
+                c.join(timeout=0.1)
+            except KeyboardInterrupt:
+                print("")
+                print("Ctrl-c received! Sending kill to threads...")
+                shutdown_event.set()
+                
+                # set loop flag true to get into loop
+                flag = True
+                while flag:
+                    #sleep 2 secs that we don't loop to often
+                    sleep(2)
+                    # set loop flag to false. If there is no thread still alive it will stay false
+                    flag = False
+                    # loop through each running thread and check if it is alive
+                    for t in threading.enumerate():
+                        # if one single thread is still alive repeat the loop
+                        if t.isAlive():
+                            flag = True
+                            
+                print("Cleaning up created indices.. "),
+                cleanup_indices(es, all_indices)
+
+    print("\nTest is done! Final results:")
+    print_stats(STARTED_TIMESTAMP)
+
+    # Cleanup, unless we are told not to
+    if not NO_CLEANUP:
+        print("Cleaning up created indices.. "),
+
+        cleanup_indices(es, all_indices)
+
+        print("Done!")  # # Main runner
+
+try:
+    main()
+
+except Exception as e:
+    print("Got unexpected exception. probably a bug, please report it.")
+    print("")
+    print(e.message)
+    print("")
+
+    sys.exit(1)
```

Comment:
instead of adding this to our repo, can we simple clone this https://github.com/logzio/elasticsearch-stress-test repo inside docker image and run it from there?

Reply:
It could be done in that way. The reason for copying these files is to make it persistent (it will be available if the upstream repo is deleted).

Reply:
all in all the data will be safe inside docker image since we won't generate a new image often

Reply:
done